File: src/shared/XXXXX.Domain/Models/Translations/ApplicationTranslation.cs
using System;

namespace XXXXX.Domain.Models
{
    public class ApplicationTranslation
    {
        public Guid Id { get; set; }
        public string LanguageCode { get; set; }
        public Guid? OrganisationTypeId { get; set; }
        public string TranslationCode { get; set; }
        public string Value { get; set; }
    }
}

File: src/app/admin/XXXXX.Admin.UI/src/domain/models/translations/applicationTranslation.ts
export class ApplicationTranslation {
    translationCode: string;
    languageCode: string;
    organisationTypeId: string | null;
    value: string;

    constructor(params: ApplicationTranslationDTO) {
        this.translationCode = params.translationCode
        this.languageCode = params.languageCode;
        this.organisationTypeId = params.organisationTypeId;
        this.value = params.value;
    }
}

export interface ApplicationTranslationDTO {
    translationCode: string;
    languageCode: string;
    organisationTypeId: string | null;
    value: string;
}

export interface UpdateApplicationTranslation {
    translationCode: string;
    languageCode: string;
    organisationTypeId: string | null;
    value: string;
}

export interface ApplicationTranslationsFilter {
    translationCode?: string;
    prefix?: string;
}